www/voll03/cv02/classes/person.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+class Person
+{
+    private string $name;
+    private string $position;
+    private DateTime $birthdate;
+    private string $phone;
+    private string $email;
+    private bool $isEmployed;
+    private string $portraitImageName;
+    private Company $company;
+
+    public function __construct(
+        string $name,

```

Comment:
![image](https://user-images.githubusercontent.com/20724910/222571640-a82a64be-5dc6-43cb-b3f4-c85fbcd5c368.png)

jako dobre cviceni ale kod je prilis redundantni... https://stitcher.io/blog/constructor-promotion-in-php-8